docs/diagnostics/PX1007.md

```diff
@@ -5,18 +5,26 @@ This document describes the PX1007 diagnostic.
 
 | Code   | Short Description                                   | Type                             | Code Fix  | 
 | ------ | ----------------------------------------------------| -------------------------------- | --------- | 
-| PX1007 | A DAC, a DAC extension, and public DAC properties should have a description in the `summary` XML tag. | Warning | Available | 
+| PX1007 | The DAC, DAC extension, or DAC property should have a description in the "summary" XML tag or "inheritdoc" XML tag | Warning | Available | 
```

Comment:
We probably should change the type to `Error` because now PX1007 will report error for mapped DAC field properties of projection DACs with incorrect annotation:
```suggestion
| PX1007 | The DAC, DAC extension, or DAC property should have a description in the "summary" XML tag or "inheritdoc" XML tag | Error | Available | 
```